src/Microsoft.ML.Transforms/Text/StopWordsRemoverTransform.cs

```diff
@@ -90,7 +90,10 @@ public enum Language
             Polish = 12,
             Czech = 13,
             Arabic = 14,
-            Japanese = 15
+            Japanese = 15,
+
+            [HideEnumValue]
+            Norwegian_Bokmal_v1 = 256
```

Comment:
>v1 [](start = 29, length = 2)

shouldn't this be v2 as the original should remain as "Norwegian_Bokmal"

Reply:
Hi @glebuk . The code as it stands is correct here. Usage of `Norwegian_Bokmal` should use whatever is best. But we must retain "old" files for backcompat, as mentioned. See [my comment here](https://github.com/dotnet/machinelearning/pull/293#issuecomment-395931937). I went into this in some slight detail.

Reply:
So should we move the previous enum id to v1 and assign a new id to the Norwegian_Bokmal then? Otherwise how would the old model be compatible?

---
In reply to: [196494433](https://github.com/dotnet/machinelearning/pull/293#discussion_r196494433) [](ancestors = 196494433)